Modules/BenchPress.Azure/Public/Confirm-AksNodePool.ps1

```diff
@@ -0,0 +1,52 @@
+﻿# INLINE_SKIP
+using module ./../Classes/ConfirmResult.psm1
+
+. $PSScriptRoot/../Private/Connect-Account.ps1
+# end INLINE_SKIP
+
+function Confirm-AksNodePool {
+  <#
+    .SYNOPSIS
+      Confirms that an AKS Node Pool exists.
+
+    .DESCRIPTION
+      The Confirm-AzBPAksNodePool cmdlet gets an AKS Node Pool using the specified AKS Node Pool and Resource Group
+      name.
+
+    .PARAMETER AKSName
```

Comment:
Doc parameter list and actual parameter list do not match.

Reply:
Done!